src/components/Book.js

```diff
@@ -1,27 +1,58 @@
 import PropTypes from 'prop-types';
 import { useDispatch } from 'react-redux';
 import { deleteBook } from '../redux/books/booksSlice';
+import './Book.css';
 
 const Book = ({ book }) => {
   const dispatch = useDispatch();
   const onRemoveClicked = (id) => {
     dispatch(deleteBook(id));
   };
   return (
-    <div>
-      <h1>{book.title}</h1>
-      <h2>
-        Author:
-        {book.author}
-      </h2>
-      <button onClick={() => onRemoveClicked(book.item_id)} type="button">Delete</button>
+    <div className="main-book" key={book.item_id}>
+      <div>
+        <p className="category">{book.category}</p>
+        <h1 className="Title">{book.title}</h1>
+        <p className="author">
+          Author:
+          {' '}
+          {book.author}
+        </p>
+        <div className="btn">
+          <button type="button" className="comments">Comments</button>
+          <div className="vertical" />
+          <button
+            type="button"
+            className="comments"
+            onClick={() => onRemoveClicked(book.item_id)}
+          >
+            Remove
+          </button>
+          <div className="vertical" />
+          <button type="button" className="comments">Edit</button>
+        </div>
+      </div>
+      <div className="circle">
+        <div className="oval" />
```

Comment:
- [ ] You've made a good addition to your project by including an oval to show progress. However, it's essential to ensure that the colors used for the oval match the design specifications closely. Please update your code to align with the design's color choices.

| Your | Design |
| --- | --- |
|  ![image](https://github.com/BilalLiaquat7/bookstore/assets/55840999/8429c2a4-555d-4c0f-b51e-06cd373de555)| ![image](https://github.com/BilalLiaquat7/bookstore/assets/55840999/08cb7098-c73d-4cfe-9346-319f34e8e438)|

By making this adjustment, you'll ensure that your progress indicator adheres to the intended design, creating a more visually consistent and appealing user interface. Keep up the great work! 🌟🎨